app/controllers/events_controller.rb

```diff
@@ -30,16 +30,34 @@ def create
     event = create_event(find_bridge)
     if event.save
       EventWorker.perform_async(event.id)
-      render json: {}, status: 202 # Accepted
+      render json: { id: event.id }, status: 202 # Accepted
     else
       render json: { error: 'Invalid parameters' }, status: 400 # Bad Request
     end
   rescue JSON::ParserError
     render json: { error: 'Invalid request. Payload must be in JSON' }, status: 400 # Bad Request
   end
 
+  def abort
+    events = if bridge_id_present
```

Comment:
Just so we are clear, this if is for aborting 1 or many events correct? If so, mind putting a little comment at the top? 1-2 lines about usage.

Reply:
KK :)

Reply:
You didn't add the text!